### DISCUSSION TITLE
CORS Error: Redirect is not allowed for a preflight request.

### DISCUSSION THREAD

**Discussion by arjunhadke7** [appwrite/appwrite#4931] [Q&A]
### 👟 Reproduction steps

When I connect to a gitpod instance using Vuejs as front end locally, I am getting the following issue in the console:

```Access to XMLHttpRequest at 'http://my-gitpod-instance.gitpod.io/v1/account' from origin 'http://localhost:8080' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.```

I considered using Gitpod since I was able to spin up a free instance.

### 👍 Expected behavior

I tried making changes to the docker compose files as per some issue links on Github but that didnt work out for me.

### 👎 Actual Behavior

<img width="1383" alt="Screenshot 2022-12-28 at 2 32 10 AM" src="https://user-images.githubusercontent.com/54884202/209722300-531e03aa-dba9-4d27-9b0f-84f1262749c6.png">

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

MacOS

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Would you please try using HTTPS for the Appwrite endpoint (your gitpod instance)? It seems like there's an HTTP redirect that doesn't work with fetch.

**Reply by arjunhadke7** [reply]
I am able to make requests from http connection as well. I made changes in the yml file at all places to change version from 1.0.2 to latest.

**Reply by stnguyen90** [reply]
> I am able to make requests from http connection as well

What do you mean? Is it working now?

> I made changes in the yml file at all places to change version from 1.0.2 to latest.

If you're trying to upgrade, you should follow the [upgrade docs](https://appwrite.io/docs/upgrade). Also, FYI, 1.2.0 should be available on https://github.com/appwrite/integration-for-gitpod now.
